frontend/src/pages/AccountSend.vue

```diff
@@ -292,21 +293,60 @@ function useSendForm() {
   }
 
   const isNativeToken = (address: string) => getAddress(address) === NATIVE_TOKEN.value.address;
-  const getMinSendAmount = (tokenAddress: string) => {
+
+  const getNativeTokenMinSendAmountFallback = (chainId: number): number => {
+    switch (chainId) {
+      case 137: return 1.0; // Polygon
+      default: return 0.01; // everything else
+    }
+  }
+
+  const getTokenMinSendAmountFallback = (
+    tokenAddress: string,
+    chainId: number
+  ): number => {
+    switch (chainId) {
+      case 137: // Polygon
+        if (tokenAddress === '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619') {
+          return 0.001; // weth token minimum
+        } else {
+          return 3; // stablecoin token minimum
+        }
+      default:
+        // Mainnet, Rinkeby, and other networks have higher ETH and stablecoin minimums
+        // due to higher fees
+        return 100
+    }
+  }
+
+  const getMinSendAmount = (tokenAddress: string): number => {
```

Comment:
Since the relayer now includes a min send amount on the returned tokens array, this method overly complicated. It seems a cleaner approach would be:
- Update `getMinSendAmount` to take the new `TokenInfoExtended` (or whatever the name is) type as input
- Refactor `wallet.ts` so the `tokens` computed property we export is of that type, and includes min send amounts
- If the relayer is down, that `tokens` export should only have ETH with the default value we define in the frontend. Otherwise, the API is working so we have all tokens and their default min send amounts, therefore we don't need to define token overrides anywhere on the frontend aside from native token overrides in `wallet.ts` (the assumption here is that we'll never get back an array of tokens without min send amounts from the API—if we currently can, then we should move the defining of token minimums to the backend so that doesn't happen, and just duplicate the native token min amount on the frontend)
- Update this method to just parse the `tokens` array for the minimum send amount and humanize it